content/kubeflow/distributed.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Kubeflow Distributed Training"
+date: 2019-11-15T00:00:00-08:00
+weight: 70
+draft: false
+---
+
+### Distributed Training using tf-operator and pytorch-operator
+
+`TFJob` is a Kubernetes [custom resource](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) that you can use to run TensorFlow training jobs on Kubernetes. The Kubeflow implementation of TFJob is in [tf-operator](https://github.com/kubeflow/tf-operator). Similarly, you can create PyTorch Job by defining a `PyTorchJob` config file and [pytorch-operator](https://github.com/kubeflow/pytorch-operator) will help create PyTorch job, monitor and keep track of the job.
+
+Go to your eks-kubeflow-workshop notebook server and browse for distributed training notebooks (eks-workshop-notebook/notebooks/03_Distributed_Training/03_01_Distributed_Training_ParameterServer.ipynb). If you haven’t installed notebook server, review fairing chapter and finish the clone the repo instructions.
```

Comment:
Go to your **eks-kubeflow-workshop** notebook server and browse for distributed training notebooks (eks-workshop-notebook/notebooks/03_Distributed_Training/03_01_Distributed_Training_ParameterServer.ipynb). If you haven’t installed notebook server, review [fairing chapter](https://eksworkshop.com/kubeflow/fairing/#create-jupyter-notebook-server) and finish the [clone the repo](https://eksworkshop.com/kubeflow/fairing/#clone-the-repo) instructions.

Reply:
Cool. I update the link and overstriking.